primitives/src/parachain.rs

```diff
@@ -328,6 +330,39 @@ impl AttestedCandidate {
 	}
 }
 
+/// A fee schedule for messages. This is a linear function in the number of bytes of a message.
+#[derive(PartialEq, Eq, PartialOrd, Hash, Default, Clone, Copy, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
+pub struct FeeSchedule {
+	/// The base fee charged for all messages.
+	pub base: Balance,
+	/// The per-byte fee charged on top of that.
+	pub per_byte: Balance,
+}
+
+impl FeeSchedule {
+	/// Compute the fee for a message of given size.
+	pub fn compute_fee(&self, n_bytes: usize) -> Balance {
+		use rstd::mem;
+		debug_assert!(mem::size_of::<Balance>() >= mem::size_of::<usize>());
```

Comment:
```suggestion
		assert!(mem::size_of::<Balance>() >= mem::size_of::<usize>());
```

If we use `debug_assert!`, it will probably never be called. On CI we execute the tests in release mode, so I don't see when this is executed in debug.

Or we could try:
https://docs.rs/static_assertions/0.3.3/static_assertions/macro.const_assert.html

As `size_of` is a const fn, it should probably work with `const_assert!`.

Reply:
It's not no-std and I don't want to have unproved panickers in the codebase.